templates/http/base/model-config.yaml

```
@@ -3,7 +3,11 @@ kind: ConfigMap
 metadata:
   name: {{values.name}}-model-config
 data:
+  {%- if values.existingModelServer %}
+  MODEL_ENDPOINT: "{{values.modelServerHost}}"
```

```suggestion
  MODEL_ENDPOINT: "{{values.modelEndpoint}}"
```

nit: since `MODEL_NAME` has its corresponding value as `values.modelName` for consistency in naming I think we should do the same for `MODEL_ENDPOINT`? wdyt?

This would bring in more changes to what the parameter in the template is called however outside of this PR.

---

Agreed. Updated.